AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialCanvas_Atom_BasicTests.py

```diff
@@ -123,7 +141,46 @@ def MaterialCanvas_BasicFunctionalityChecks_AllChecksPass():
             Tests.material_graph_name_is_test1,
             material_graph_name == "test1")
 
-        # 8. Look for errors and asserts.
+        # 8. Create a new material_graph from material_graph_document_ids[0].
+        material_graph = material_canvas_utils.get_graph(material_graph_document_ids[0])
+        material_graph_id = material_canvas_utils.get_graph_id(material_graph_document_ids[0])
+        Report.result(
+            Tests.material_graph_created,
+            material_graph.typename == "AZStd::shared_ptr<Graph>" and material_graph is not None)
+
+        # 9. Create a new world_position_node in memory.
+        world_position_node = material_canvas_utils.create_node_by_name(material_graph, "World Position")
+        Report.result(
+            Tests.world_position_node_created,
+            world_position_node.typename == "AZStd::shared_ptr<Node>" and world_position_node is not None)
```

Comment:
we really need an ebus call to get the type of node based on a node AZStd::shared_ptr<Node>
did this call create the node I wanted or something else? I just know it's return is type AZStd::shared_ptr<Node>
these bus calls behind this don't have any error checking they don't use AZ_Outcome.IsSuccess() we just trust that pointers are valid and belong to <Node> on my graph or that are valid

Reply:
it will fail if we don't specify the correct pointer for the graph and node because it will return None which will fail on the `.typename` check

the `create_node_by_name()` call creates the node in memory first, it has to be added to the graph using `add_node()` after creating one in memory